conf/routes

```diff
@@ -10,9 +10,6 @@ GET         /oauthCallback                           controllers.PandaAuthContro
 GET         /logout                                  controllers.PandaAuthController.logout
 GET         /login/status                            controllers.PandaAuthController.status
 
-# healthcheck
-GET         /status                                  controllers.StatusController.healthStatus
```

Comment:
https://github.com/guardian/editorial-tools-platform/pull/305 needs to go out first